source/api_documentation/conversion_reporting.rst

```diff
@@ -1,11 +1,29 @@
-Conversion Reporting
+Conversion Reporting (Deprecated)
 ====================
 
+.. raw:: html
+
+  <div style="margin-bottom: 20px; padding: 15px; background-color: #fff3cd; color: #333;"><p>
+    <b>Notice:</b>
+    <a href="https://developers.invoca.net/en/latest/api_documentation/signal_api/index.html">Signal API</a> is now the supported way for reporting conversions on calls.
```

Comment:
> now the supported way for reporting conversions

Conversions themselves are being deprecated in favor of signals right? Would it be clearer to omit _conversions_ from this to prevent confusion? Or would that be more confusing?

Reply:
My concern was just someone reading that thinking we are continuing to support "Conversion". I think its fine as is if you don't think that's an issue.

Reply:
Less chance for confusion sounded good to me, so I updated the message: fd67511